spec/requests/partnerships_spec.rb

```diff
@@ -58,6 +58,63 @@
         get "/partnerships/bronze-sponsor"
         expect(response.body).to include("This subscription will renew every month")
       end
+
+      context "when sponsorship exists" do
+        let(:org) { create(:organization) }
+
+        before do
+          create(:organization_membership, user: user, organization: org, type_of_user: "admin")
+          Credit.add_to_org(org, 1000)
+          sign_in user
+        end
+
+        describe "level sponsorshi[s" do
+          # should pass
```

Comment:
should :D ?

Reply:
Haha, these were notes for when I started working on the tests for the initial implementation (so some should have failed). I'll remove these notes :)